core/unit_test/incremental/Test12b_TeamScratch.hpp

```diff
@@ -111,6 +111,9 @@ TEST(TEST_CATEGORY, IncrTest_12b_TeamScratch) {
   test.run(4, 7, 10);
 #ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
This might lead to merge conflicts with #3748 . Both the scratch related incremental tests are being updated there too with respect to their problem sizes. So one of us might have to wait until the other one gets merged.

Reply:
I am used to having merge conflicts after almost every of my pull requests being merged. :slightly_smiling_face: No big deal for me to rebase and fix the conflicts if necessary.